### PR TITLE
adding bugsnag logging of errors

### PR DESCRIPTION
**Pull request by speizerj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .DS_Store
+.idea/
```
Comment (zackbloom) [inline .gitignore]: You may want to add this to your general gitignore file.

Do `git config --global core.excludesfile ~/.gitignore`, and then place it in `~/.gitignore`

File: includes/settings.php [hunk 2]
```diff
@@ -122,6 +122,8 @@ function eager_save_login_callback() {
 }
 
 function eager_exec_in_background() {
+    global $bugsnag;
```
Comment (zackbloom) [inline includes/settings.php]: Are we worried about this conflicting with someone who has their own Bugsnag install?

Comment (zackbloom) [inline includes/settings.php]: For example, what if they have this installed? https://bugsnag.com/docs/notifiers/wordpress

File: .gitignore (hunk 1 above)
Comment (speizerj) [inline .gitignore]: That is super helpful, thanks :rocket:

File: includes/settings.php (hunk 2 above)
Comment (speizerj) [inline includes/settings.php]: Yep good call. What if we make the global something longish and specific, like eager_wordpress_bugsnag?

Comment (zackbloom) [inline includes/settings.php]: `$eagerBugsnag` is probably fine.

**Comment by speizerj** [comment]
@zackbloom this is ready to go if you want to take a look

**Pull request closed without merging**